circleci note added (#3001)
SumoLogic/sumologic-documentation: docs/observability/sdo/set-up-sdo/sdo-manual-configuration.md

1. Under this collector, create the following two [http sources](/docs/send-data/hosted-collectors/http-source/logs-metrics):
   * `_sourcecategory=circleci/job-collector`
   * `_sourcecategory="circleci/workflow-collector"`

>    Sumo Logic sources by default have [multiline processing](https://help.sumologic.com/docs/send-data/reference-information/collect-multiline-logs/) enabled. You need to disable it here for structured log collection of CircleCI.

Reviewer: ```suggestion
   Sumo Logic sources, by default, have [multiline processing](https://help.sumologic.com/docs/send-data/reference-information/collect-multiline-logs/) enabled. You'll need to disable it here in order to collect structured logs for CircleCI.
```
